<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCorpusRepositoryWizard</class>
 <widget class="QWizard" name="NewCorpusRepositoryWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <widget class="QWizardPage" name="pageDatastoreType">
   <property name="title">
    <string>New Corpus</string>
   </property>
   <property name="subTitle">
    <string>This wizard will guide you through the process of creating a new Praaline corpus.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBoxBasic">
      <property name="title">
       <string>Basic information</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelCorpusID">
         <property name="text">
          <string>Corpus ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editCorpusID"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelCorpusName">
         <property name="text">
          <string>Corpus Name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editCorpusName"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBoxDatastoreType">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Corpus storage method</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7" columnstretch="1,0">
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
         <item row="2" column="1">
          <widget class="QRadioButton" name="optionRemoteDB">
           <property name="text">
            <string>Connect to a database server (workgroup mode)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QLabel" name="iconLocalDb">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../praaline.qrc">:/icons/corpusexplorer/corpus_local_database.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="labelRemoteDb">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Corpus metadata and annotation data are stored on a database server on the network (MySQL or Postgres). Media files are stored in folders (including network folders) and accessed from there. Multiple users may connect to the same corpus and work on it simultaneously.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="indent">
            <number>18</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelLocalDb">
           <property name="text">
            <string>Corpus metadata and annotation data will be stored on a local SQLite database. Media files are stored filesystem folders. This is the default mode for individual projects.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="indent">
            <number>18</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0" rowspan="2">
          <widget class="QLabel" name="iconFiles">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../praaline.qrc">:/icons/corpusexplorer/corpus_file_collection.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QRadioButton" name="optionFiles">
           <property name="text">
            <string>File-based storage (not recommended)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="optionLocalDB">
           <property name="text">
            <string>Database stored on my computer (solo project mode)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="labelFiles">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Corpus metadata and annotations are stored in XML files, and media files in folders. You will not be able to use Praaline queries and the concordancer; some plugins may also not work. Use this mode if you just want to organise a collection of files.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="indent">
            <number>18</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" rowspan="2">
          <widget class="QLabel" name="iconRemoteDb">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../praaline.qrc">:/icons/corpusexplorer/corpus_remote_database.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="labelDatastoreType">
         <property name="text">
          <string>Praaline supports many different ways to store and organise your corpus data. Please select the type of corpus you want to create:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="pageLocalDB">
   <property name="title">
    <string>New Corpus - Local database</string>
   </property>
   <property name="subTitle">
    <string>In this step, you may complete the information necessary to create a new Corpus as a local database. You may also select templates to help you get started with the structure of its metadata and annotations.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBoxDatastores">
      <property name="title">
       <string>Where will the corpus data be stored?</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="labelBaseFolder">
         <property name="text">
          <string>Corpus files in folder:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editBaseFolder"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="commandSelectFolder">
         <property name="text">
          <string>Select folder</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="optionSameDatastore">
         <property name="text">
          <string>Save metadata and annotations in the same file</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelMetadataDatabase">
         <property name="text">
          <string>Metadata database:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editMetadataDatabase"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelAnnotationDatabase">
         <property name="text">
          <string>Annotation database:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editAnnotationDatabase">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBoxMetadataTemplates">
      <property name="title">
       <string>Metadata structure templates</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTreeView" name="treeviewMetadataTemplates"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBoxAnotationTemplates">
      <property name="title">
       <string>Annotation structure templates</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTreeView" name="treeviewAnnotationTemplates"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editCorpusID</tabstop>
  <tabstop>editCorpusName</tabstop>
  <tabstop>optionLocalDB</tabstop>
  <tabstop>optionRemoteDB</tabstop>
  <tabstop>optionFiles</tabstop>
  <tabstop>editBaseFolder</tabstop>
  <tabstop>commandSelectFolder</tabstop>
  <tabstop>optionSameDatastore</tabstop>
  <tabstop>editMetadataDatabase</tabstop>
  <tabstop>editAnnotationDatabase</tabstop>
  <tabstop>treeviewMetadataTemplates</tabstop>
  <tabstop>treeviewAnnotationTemplates</tabstop>
 </tabstops>
 <resources>
  <include location="../praaline.qrc"/>
 </resources>
 <connections/>
</ui>
